### ISSUE TITLE
[FR] Anonymous installs for MS Store

### ISSUE THREAD

**Issue by Kein** [microsoft/winget-cli#1726]
### Description of the new feature / enhancement

Currently, installing anything, even free apps, from MS Store has a hard requirement of an account:
```
Found Windows Calculator [9WZDNCRFHVN5] Version Unknown
...
Verifying/Requesting package acquisition...
Verifying/Requesting package acquisition failed: no store account found
```

There is no reason these apps that aren't tired to any account-based subscription should be still gated behind account, i.e. `apt-get` or `yum` dont require an account just to fetch python.

### Proposed technical implementation details

The actual content of free apps does not require any account tokens to be accessed. For example, if you copy the link for `Windows Calculator` app:
https://www.microsoft.com/en-us/p/windows-calculator/9wzdncrfhvn5
and use link-generator for MS store
https://store.rg-adguard.net
you will get temporal links to download not just the app package but all dependencies as well. They are not gated by any means. Therefore, same mechanism can be be used for any anonymous installs of free apps with winget.

**Comment by vedantmgoyal2009** [comment]
Duplicate of #1585

**Comment by jedieaston** [comment]
Duplicate of #1585

**Comment by BentramLawrence** [comment]
shamefully closing stuff that isnt fixed
